src/lib/es2023.array.d.ts

```diff
@@ -39,11 +39,11 @@ interface Array<T> {
     toSorted(compareFn?: (a: T, b: T) => number): T[];
 
     /**
-     * Copies an array and removes elements and, if necessary, inserts new elements in their place. Returns the copied array.
+     * Copies an array and removes elements while, if necessary, inserting new elements in their place, returning the remaining elements.

```

Comment:
The description already says how the copy is mutated, so I find the “remaining elements” language extremely confusing.